internal/processmon/processmon.go

```diff
@@ -205,34 +213,26 @@ func (p *ProcessMon) pollStdOutAndErr(cmd *exec.Cmd, exited chan int, contextID
 }
 
 // getLaunchProcessCmd returns the command used to launch the enforcerd
-func (p *ProcessMon) getLaunchProcessCmd(arg string, contextID string) *exec.Cmd {
+func (p *processMon) getLaunchProcessCmd(arg string, contextID string) *exec.Cmd {
 
 	cmdName, _ := osext.Executable()
 	cmdArgs := []string{arg}
 
-	if _, ok := GlobalCommandArgs["--log-level-remote"]; ok {
-		cmdArgs = append(cmdArgs, "--log-level")
-		cmdArgs = append(cmdArgs, GlobalCommandArgs["--log-level-remote"].(string))
-	}
-
-	if _, ok := GlobalCommandArgs["--log-to-console"]; ok && GlobalCommandArgs["--log-to-console"].(bool) {
-		cmdArgs = append(cmdArgs, "--log-to-console")
-	} else {
-		cmdArgs = append(cmdArgs, "--log-id")
-		cmdArgs = append(cmdArgs, contextID)
-	}
+	cmdArgs = append(cmdArgs, p.launcProcessArgs...)
 
 	zap.L().Debug("Enforcer executed",
 		zap.String("command", cmdName),
 		zap.Strings("args", cmdArgs),
 	)
+
 	return exec.Command(cmdName, cmdArgs...)
 }
 
-func (p *ProcessMon) getLaunchProcessEnvVars(procMountPoint string, contextID string, randomkeystring string, statsServerSecret string, refPid int, refNSPath string) []string {
+// getLaunchProcessEnvVars returns a slice of env variable strings where each string is in the form of key=value
+func (p *processMon) getLaunchProcessEnvVars(procMountPoint string, contextID string, randomkeystring string, statsServerSecret string, refPid int, refNSPath string) []string {
 
 	mountPoint := "APORETO_ENV_PROC_MOUNTPOINT=" + procMountPoint
-	namedPipe := "APORETO_ENV_SOCKET_PATH=/var/run/" + contextID + ".sock"
+	namedPipe := "APORETO_ENV_SOCKET_PATH=" + contextID2SocketPath(contextID)
 	statsChannel := "STATSCHANNEL_PATH=" + rpcwrapper.StatsChannel
```

Comment:
Why some are prefix with APORETO_ and others not?

Reply:
fixed